torchvision/datapoints/_datapoint.py

```diff
@@ -60,27 +66,29 @@ def __torch_function__(
         2. For most operations, there is no way of knowing if the input type is still valid for the output.
 
         For these reasons, the automatic output wrapping is turned off for most operators. The only exceptions are
-        listed in :attr:`Datapoint._NO_WRAPPING_EXCEPTIONS`
+        listed in _FORCE_TORCHFUNCTION_SUBCLASS
```

Comment:
Note: the docstring above is mostly wrong / obsolete now. If this is merged I would rewrite everything. Same for a lot of the comments.

Reply:
I think I will be able to write better quality comments / docstring once I start writing the user-facing docs.